src/Artsy/SystemContext.tsx

```diff
@@ -24,6 +24,7 @@ export interface SystemContextProps {
    * FIXME: Ask alloy how to pass one-off props like this in from force
    */
   notificationCount?: number
+  searchQuery?: string
```

Comment:
I need to follow up this week and get proper app-level interfaces in that extend from SystemContext, and can be imported into force as a contract.

Reply:
How does `searchQuery` get updated? I see it defined here, and then read [in `SearchBar.tsx`](https://github.com/artsy/reaction/pull/2495/files#diff-195cf326e4088c5f9ff2c6d47cc4248cR424), but it's not obvious to me how that value ever changes in the `SystemContext`. I'm certain it's me not understanding - can you explain?

Reply:
Yup! It's passed in from the server, in the companion PR [here](https://github.com/artsy/force/pull/4149/files#diff-3df7ffd25fef9729a61746506a33251bR39).